server/src/services/rename/onRename.ts

```diff
@@ -25,19 +21,7 @@ export const onRename = (serverState: ServerState) => {
         serverState,
         "onRenameRequest",
         params.textDocument.uri,
-        (documentAnalyzer) =>
-          rename(documentAnalyzer, params.position, params.newName)
-      );
-
-      // Renames are multifile, if the change to the current
-      // editor goes to validation before any file changes
-      // are recorded, preprocessing won't recognise that
-      // the cache is no longer valid, hence we clear it
-      // before returning
-      await invalidateWorkerPreprocessCache(
```

Comment:
What is the thinking on removing this, is the case no longer needed or covered in another way?

Reply:
It was no longer needed because the strategy was switched to "replace sources with openDocuments", which works nicely but yesterday I found out it doesn't work on Windows. So I brought back the invalidate approach with the `project.invalidateBuildCache()` call